package.json

```diff
@@ -42,8 +42,8 @@
     "jss": "10.0.0",
     "lodash": "4.17.15",
     "prop-types": "15.7.2",
-    "react-markdown": "4.1.0",
     "react-is": "^16.10.2",
+    "react-markdown": "4.1.0",
```

Comment:
why

Reply:
Dont know. I was testing bootstrap before.

Reply:
I was more commenting on that you removed `"react-markdown": "4.1.0",` and added it again 👍 Doesn't matter really. Just found it funny